test/srt/test_qwen2_vl_openai_server.py

```diff
@@ -0,0 +1,294 @@
+import base64
+import io
+import json
+import os
+import unittest
+
+import numpy as np
+import openai
+import requests
+from decord import VideoReader, cpu
+from PIL import Image
+
+from sglang.srt.utils import kill_child_process
+from sglang.test.test_utils import (
+    DEFAULT_TIMEOUT_FOR_SERVER_LAUNCH,
+    DEFAULT_URL_FOR_TEST,
+    popen_launch_server,
+)
+
+
+class TestQWen2VLServer(unittest.TestCase):
```

Comment:
Good test! Instead of copying this over. Is it possible to reuse code in the existing `test_vision_openai_server.py`? A few methods:
1. Write a for loop in `test_vision_openai_server.py` to loop over multiple models.
2. Let `TestQWen2VLServer` be a child class of `TestOpenAIVisionServer`

Reply:
done